schedule/schedule-appengine/src/main/java/com/jasify/schedule/appengine/mail/DefaultMailService.java

```diff
@@ -124,22 +146,44 @@ private Message createMessage(InternetAddress fromAddress, InternetAddress[] toA
         }
 
         message.setSubject(subject);
+        message.setSentDate(new Date());
 
-        Multipart mp = new MimeMultipart();
+        /*
+         * The section 5.1.4 of RFC 2046 defines multipart/alternative
+         * http://tools.ietf.org/html/rfc2046#section-5.1.4
+         * ... in general the LAST part is the best choice ...
+         */
+        MimeMultipart alternative = new MimeMultipart("alternative");
+
+        if (textBody != null) {
+            MimeBodyPart textPart = new MimeBodyPart();
+            textPart.setContent(textBody, "text/plain");
```

Comment:
org.apache.http.entity.ContentType.TEXT_PLAIN and org.apache.http.entity.ContentType.TEXT_HTML